lib/src/component/aria_mixin.dart

```diff
@@ -41,6 +42,8 @@ abstract class $AriaPropsMixin {
 abstract class _$AriaPropsMixin {
   Map get props;
 
+  ///See: <https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes>
```

Comment:
This comment will get attached to `activedescendant`; could we move it to the doc comment above the `_ $AriaPropsMixin` class?